### PR TITLE
@86eqcxfwc-Added Program management in web back office

### PR DESCRIPTION
**Pull request by emmanlemon**
[WIP] Added UI in Program Management
image

### CODE REVIEW

**Review by JMSevilla — APPROVED** [review]

File: apps/web-customer/src/components/blocks/HubBlocks/ProgramListBlock/ProgramGridView.tsx [hunk 1]
```diff
@@ -12,10 +12,10 @@ import {
   CompletedWhiteIcon,
 } from "core-library/assets";
 import { StandardProgramListType } from "core-library/types/wc/programList";
-import useCalculateProgramProgress from "../../../../core/hooks/useCalculateProgramProgress";
 import { ProgressCircle } from "../../../../components/ProgressCircle/ProgressCircle";
 import { DialogBox } from "core-library/components/Dialog/DialogBox";
 import { ProgramGridContent } from "./ProgramGridContent";
+import useCalculateProgramProgress from "core-library/core/hooks/useCalculateProgramProgress";
```
Comment (DropitupQwerty) [inline apps/web-customer/src/components/blocks/HubBlocks/ProgramListBlock/ProgramGridView.tsx:18]: directly accessing the hooks from the file 

import the custom hook inside the index.ts in hooks directory

File: packages/core-library/components/AccordionList/AccordionList.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  Accordion,
+  AccordionSlots,
+  Fade,
+  IconButton,
+} from "@mui/material";
+import { StandardProgramListType } from "core-library/types/wc/programList";
+import { CustomAccordionSummary } from "./AccordionSummary";
+import { CustomAccordionDetails } from "./AccordionDetails";
+import { styled } from "@mui/material";
+import useCalculateProgramProgress from "../../core/hooks/useCalculateProgramProgress";
```
Comment (DropitupQwerty) [inline packages/core-library/components/AccordionList/AccordionList.tsx:18]: same comment above

File: packages/core-library/components/AccordionList/AccordionList.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+import EditIcon from "@mui/icons-material/Edit";
+import DeleteIcon from "@mui/icons-material/Delete";
```
Comment (DropitupQwerty) [inline packages/core-library/components/AccordionList/AccordionList.tsx:20]: Can we use the evaicons here?

File: packages/core-library/components/AccordionList/AccordionList.tsx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+interface Props {
+  program: StandardProgramListType[];
+  isOptions?: boolean;
+}
+
+const StyledAccordion = styled(Accordion)(({ expanded }) => ({
```
Comment (DropitupQwerty) [inline packages/core-library/components/AccordionList/AccordionList.tsx:27]: Isolate this StyledAccordion in a content folder

File: packages/core-library/system/app/internal/blocks/Hub/Program/ProgramManagementBlock.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Property of the NCLEX Power.
+ * Reuse as a whole or in part is prohibited without permission.
+ * Created by the Software Strategy & Development Division
+ */
+import { Box, Container, Typography } from "@mui/material";
+import { standardProgramList } from "./constant/ProgramConstants";
+import { Button } from "../../../../../../components";
+import { AccordionList } from "../../../../../../components/AccordionList/AccordionList";
```
Comment (DropitupQwerty) [inline packages/core-library/system/app/internal/blocks/Hub/Program/ProgramManagementBlock.tsx:9]: Directly accessing the component import the component in index.ts

File: packages/core-library/system/app/internal/blocks/Hub/Program/ProgramManagementBlock.tsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+          }}
+        >
+          <Typography
+            sx={{ fontSize: "1.5rem", fontWeight: "bold", color: "#560bad" }}
+          >
+            Program Management List
+          </Typography>
+          <Button sx={buttonStyle}>Create</Button>
+        </Box>
+        <AccordionList program={standardProgramList} isOptions={true}/>
```
Comment (DropitupQwerty) [inline packages/core-library/system/app/internal/blocks/Hub/Program/ProgramManagementBlock.tsx:38]: you can drop the ={true}

<AccordionList program={standardProgramList} isOptions/>

**Pull request closed without merging**